poem_plugins/config/__init__.py

```diff
@@ -0,0 +1,28 @@
+from enum import unique
+
+from pydantic import BaseModel, Field
```

Comment:
the general problem is that it's an incredibly heavy thing to check some parameters. Without it it will be a couple of lines of code more, but much better IMHO.

Reply:
yep, trying to avoid using pydantic with nested dataclass with simple  validation